huggingface/pytorch/buildspec-neuron.yml

```diff
@@ -2,8 +2,8 @@ account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 base_framework: &BASE_FRAMEWORK pytorch
 framework: &FRAMEWORK !join [ "huggingface_", *BASE_FRAMEWORK]
-version: &VERSION 1.9.1
-short_version: &SHORT_VERSION 1.9
+version: &VERSION 1.10.1
```

Comment:
Can you please check the version. It has to be 1.10.2. For neuron sdk 1.19.1 only torch-neuron-1.10.2 is released. Please check the following [link](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/releasecontent.html#neuron-release-content)